frontend/src/concepts/design/vars.scss

```
@@ -4,94 +4,129 @@
   --ai-cluster-storage--BorderColor: var(--ai-organize--BorderColor);
   --ai-cluster-storage--SelectedColor: var(--ai-organize--SelectedColor);
   --ai-cluster-storage--Color: var(--ai-organize--BorderColor);
+  --ai-cluster-storage--IconColor: var(--ai-organize--IconColor);
 
   --ai-notebook--BackgroundColor: var(--ai-training--BackgroundColor);
   --ai-notebook--LabelBackgroundColor: var(--ai-training-LabelBackgroundColor);
   --ai-notebook--BorderColor: var(--ai-training--BorderColor);
   --ai-notebook--SelectedColor: var(--ai-training--SelectedColor);
   --ai-notebook--Color: var(--ai-training--Color);
+  --ai-notebook--IconColor: var(--ai-training--IconColor);
 
   --ai-model--BackgroundColor: var(--ai-serving--BackgroundColor);
   --ai-model--LabelBackgroundColor: var(--ai-serving-LabelBackgroundColor);
   --ai-model--BorderColor: var(--ai-serving--BorderColor);
   --ai-model--SelectedColor: var(--ai-serving--SelectedColor);
   --ai-model--Color: var(--ai-serving--BorderColor);
+  --ai-model--IconColor: var(--ai-serving--IconColor);
 
   --ai-pipeline--BackgroundColor: var(--ai-training--BackgroundColor);
   --ai-pipeline--LabelBackgroundColor: var(--ai-training-LabelBackgroundColor);
   --ai-pipeline--BorderColor: var(--ai-training--BorderColor);
   --ai-pipeline--SelectedColor: var(--ai-training--SelectedColor);
   --ai-pipeline--Color: var(--ai-training--BorderColor);
+  --ai-pipeline--IconColor: var(--ai-training--IconColor);
 
   --ai-model-server--BackgroundColor: var(--ai-serving--BackgroundColor);
   --ai-model-server--LabelBackgroundColor: var(--ai-serving-LabelBackgroundColor);
   --ai-model-server--BorderColor: var(--ai-serving--BorderColor);
   --ai-model-server--SelectedColor: var(--ai-serving--SelectedColor);
   --ai-model-server--Color: var(--ai-serving--Color);
+  --ai-model-server--IconColor: var(--ai-serving--IconColor);
 
   --ai-user--BackgroundColor: var(--ai-set-up--BackgroundColor);
   --ai-user--LabelBackgroundColor: var(--ai-set-up-LabelBackgroundColor);
   --ai-user--BorderColor: var(--ai-set-up--BorderColor);
   --ai-user--SelectedColor: var(--ai-set-up--SelectedColor);
   --ai-user--Color: var(--ai-set-up--BorderColor);
+  --ai-user--IconColor: var(--ai-set-up--IconColor);
 
   --ai-group--BackgroundColor: var(--ai-set-up--BackgroundColor);
   --ai-group--LabelBackgroundColor: var(--ai-set-up-LabelBackgroundColor);
   --ai-group--BorderColor: var(--ai-set-up--BorderColor);
   --ai-group--SelectedColor: var(--ai-set-up--SelectedColor);
   --ai-group--Color: var(--ai-set-up--BorderColor);
+  --ai-group--IconColor: var(--ai-set-up--IconColor);
+}
+:root:where(.pf-v6-theme-dark) {
+  --ai-set-up--BackgroundColor: #CA6C0F;
+  --ai-set-up--IconColor: #f2f2f2;
+
+  --ai-config--BackgroundColor: #CA6C0F;
+  --ai-config--IconColor: #f2f2f2;
+
+  --ai-organize--BackgroundColor: #CA6C0F;
+  --ai-organize--IconColor: #f2f2f2;
+
+  --ai-training--BackgroundColor: #37A3A3;
```

Should we not be referencing the color variables instead of the values themselves?
```suggestion
  --ai-training--BackgroundColor: var(--pf-t--color--teal--50);
```